src/Action.hh

```diff
@@ -179,7 +186,10 @@ public:
         return getParamI(0);
     }
     inline int getParamI(uint n) const {
-        return n < _i.size() ? _i[n] : 0;
+        return n < _i.size() ? _i[n].first : 0;
+    }
+    inline ActionParamUnit getParamUnit(uint n) const {
```

Comment:
Even though this gives better type safety, just use an plain old enum and set the second parameter to the unit.

Should save some memory and not confuse the use of I parameters whenever they are not of a unit type.

The whole using a standard Action type for all actions is what I think should be changed in the future (not this PR) to get a good description of the arguments available when executing the action.

Reply:
Aha! I didn't think of using setParamI like that. Fixed.